Makefile

```diff
@@ -32,11 +32,17 @@ ifeq ($(PYTHON),)
 endif
 PYTHON_BIN := $(shell which $(PYTHON))
 PYTHON_VER := $(word 2,$(shell $(PYTHON) -V 2>&1))
+GO_MIN_VER := 1.5
```

Comment:
Were you able to confirm that it works with 1.6? Be conservative and require 1.7 if not.